src/webrtc/peerconnection.ts

```diff
@@ -536,7 +479,7 @@ export class PeerConnectionClass implements PeerConnection<signals.Message> {
     // Listen for heartbeats from the other side.
     var lastPingTimestamp :number = Date.now();
     channel.dataFromPeerQueue.setSyncHandler((data:Data) => {
-      if (data.str && data.str === PeerConnectionClass.HEARTBEAT_MESSAGE_) {
+      if (data.str && data.str === HEARTBEAT_MESSAGE_) {
```

Comment:
`undefined === HEARTBEAT_MESSAGE_` is safe and evaluates to false, so you don't need the first part of this.